### PR TITLE
21.7 fb e iacuc add columns

### PR DESCRIPTION
**Pull request by jonesgaohsu**
#### Rationale
Update of schema for ONPRC_ehr.eIACUC Protocols to add fields from eIACUC 2

#### Related Pull Requests
*

### CODE REVIEW

**Comment by jonesgaohsu** [comment]
Oko need some guidance, some how this is reading an update to onprc_billing and the commit was only to include updates to onprc_ehr files.  Should I kill this request (and if so how) and resubmit

File: onprc_ehr/resources/etls/HansenQueriesToFile.xml [hunk 1]
```diff
@@ -1,92 +1,105 @@
 <?xml version="1.0" encoding="UTF-8" ?>
+<etl xmlns="http://labkey.org/etl/xml">
+21.7_fb_HansenQuerytoFileXML
```
Comment (Ohsudev) [inline onprc_ehr/resources/etls/HansenQueriesToFile.xml:3]: There seems to be a missing xml tag here.

File: onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql [hunk 2]
```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
+--Update to use corredct method to cross container
+--jonesga 11/5/2021
+=======
```
Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:4]: Please remove this string, or add a sql comment notation if needed.

File: onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql [hunk 3]
```diff
@@ -1,4 +1,9 @@
+<<<<<<< HEAD
```
Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:1]: Please remove this string, or add a sql comment notation if needed.

File: onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql [hunk 4]
```diff
@@ -1,4 +1,9 @@
...
+--Update to use corredct method to cross container
+--jonesga 11/5/2021
+=======
 
+>>>>>>> origin/release21.7-SNAPSHOT
```
Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:6]: Please remove this string, or add a sql comment notation if needed.

File: onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql [hunk 5]
```diff
@@ -8,6 +13,10 @@ SELECT d.rowid,
        d.assaytype,
        d.container,
        d.totalTests
+<<<<<<< HEAD
```
Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:16]: Please remove this string, or add a sql comment notation if needed.

Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:16]: Please remove this string, or add a sql comment notation if needed.

File: onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql [hunk 6]
```diff
@@ -8,6 +13,10 @@ SELECT d.rowid,
...
+FROM  StudyDetails_RandalData s, "/ONPRC/Core Facilities/Genetics Core/MHC_Typing".geneticscore.mhc_data d
+=======
```
Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:18]: Please remove this string, or add a sql comment notation if needed.

File: onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql [hunk 7]
```diff
@@ -8,6 +13,10 @@ SELECT d.rowid,
...
 
 FROM  StudyDetails_RandalData s, Site.{substitutePath moduleProperty('ONPRC_EHR','MHC_Container')}.geneticscore.mhc_data d
+>>>>>>> origin/release21.7-SNAPSHOT
```
Comment (Ohsudev) [inline onprc_ehr/resources/queries/onprc_ehr/RandalData_MHCData.sql:21]: Please remove this string, or add a sql comment notation if needed.

**Comment by jonesgaohsu** [comment]
Will be updated in 22.3

**Pull request closed without merging**